README.md

```diff
@@ -6,12 +6,22 @@ This repository contains python scripts to construct the Open Macromolecular Gen
 <img src="https://github.com/TheJacksonLab/OpenMacromolecularGenome/blob/main/data/figure/schematic_diagram.jpg" width="500" height="500">
 </p>
 
+# Setup
 ## Set up Python environment with Anaconda 
 ```
 conda env create -f environment.yml
 ``` 
+## eMolecules Database
+To begin, you will need the `version.smi` file from eMolecules.
+Download here: https://www.emolecules.com/structure_download_june_2023
```

Comment:
Thanks for adding information about the eMolecules database! I would like to change the address to "https://marketing.emolecules.com/incremental-file-download" not to specify the data version.  
```suggestion
Download here: https://marketing.emolecules.com/incremental-file-download
```